tests/Routing/RoutingUrlGeneratorTest.php

```diff
@@ -128,7 +128,7 @@ public function testControllerRoutesWithADefaultNamespace()
         $route = new Illuminate\Routing\Route(['GET'], 'foo/bar', ['controller' => 'namespace\foo@bar']);
         $routes->add($route);
 
-        $route = new Illuminate\Routing\Route(['GET'], 'something/else', ['controller' => 'something\foo@bar']);
+        $route = new Illuminate\Routing\Route(['GET'], 'something/else', ['controller' => '\something\foo@bar']);
```

Comment:
The testcase wasn't correct in the first place. Can you define "App\Test\Foo@bar" as a route in laravel, and have it working correctly?

It will look for ```App\Http\Controller\App\Test\Foo@bar``` and you will get a class not found exception.